gwpy/frequencyseries/tests/test_frequencyseries.py

```diff
@@ -228,6 +228,12 @@ def test_to_from_pycbc(self, array):
         a2 = type(array).from_pycbc(array.to_pycbc(copy=False), copy=False)
         assert shares_memory(array.value, a2.value)
 
+        # test conversion when f0 != 0
+        array.f0 = 1.
+        with pytest.raises(ValueError) as exc:
+            array.to_pycbc()
+        assert str(exc.value).startswith("Cannot convert FrequencySeries")
```

Comment:
I think the message matching can be handled directly by `pytest.raises`:

```suggestion
        with pytest.raises(
            ValueError,
            match="^Cannot convert FrequencySeries",
        ):
            array.to_pycbc()
```

(this is not followed universally in other locations in GWpy)